hack/check.sh

```diff
@@ -32,6 +32,14 @@ check() {
     echo "CHECK: go vet, check code syntax"
     packages=$(go list ./... | sed 's/^_//')
     echo "${packages}" | xargs go vet 2>&1
+
+    # boilerplate check
+    echo "CHECK: boilerpalte, check code boilerplate"
+    result=$(git ls-files | xargs go run ./hack/boilerplate/check-boilerplate.go)
```

Comment:
```
Dragonfly (new-807) $ git ls-files | xargs go run ./hack/boilerplate/check-boilerplate.go
Dragonfly (new-807) $ echo $?
0
version (new-807) $ vi version/version.go
Dragonfly (new-807) $ git ls-files | xargs go run ./hack/boilerplate/check-boilerplate.go
error validating "version/version.go": the file is missing a boilerplate
exit status 1
```
It works with the check file.

While I am afraid that `make check` fails since there is no docker daemon installed on my machine. I think `make file` should be friendly with developer's local machine, like windows, mac and so on. And there is no need to depend on the docker engine.

Reply:
Yes. I am also curious whether the `./hack/check-docker.sh` can be stripped out from `make check`.
But `hack/check.sh` is OK, right?
```Makefile
check:
	@echo "Begin to check code formats."
	./hack/check.sh
	@echo "Begin to check dockerd whether is startup"
	./hack/check-docker.sh
.PHONY: check
```

Reply:
> whether the ./hack/check-docker.sh can be stripped out from `make check`.

I think we should strip that from `make check`. 

`hack/hack.sh` is OK.